src/components/navbar/navbar.import.less

```diff
@@ -1,72 +1,69 @@
-@primary: #000;
-@secondary: #FFF;
-
+@primary: #F7F7F7;
+@secondary: #000;
```

Comment:
@nicholasmiller1 It's good to use variables in LESS, but we should keep them in a `variables.import.less` file in our `/config`, so we can use the same variables throughout the project. 

We will also want to name them with more descriptive names: `light-grey` instead of `secondary`, etc. 

Finally, we'll need a naming convention that differentiates between different shades of the same color. I use `light-grey-1`, `light-grey-2`, etc. (2 is darker than 1)